### ISSUE TITLE
Godot Editor crash when referencing node in Autoloaded scene script.

### ISSUE THREAD

**Issue by jeffgamedev** [godotengine/godot#68901]
### Godot version

v4.0.beta5.official [89a33d28f]

### System information

Windows 10

### Issue description

When referencing an onready node within the GDScript editor on a script attached to an autoloaded scene, the editor crashes.

### Steps to reproduce

- Create an autoloaded Control scene.
- Add a child Control node.
- Create and attach a script to the autoloaded Control node that extends Control.
- Create an onready var that references the child node.
- Access the onready var child node via the GDScript editor in a method.
- Observe the editor crashes.

### Minimal reproduction project

[reference-test-crash.zip](https://github.com/godotengine/godot/files/10047985/reference-test-crash.zip)

In my reproduction project, simply open the "testui.gd", and in the _ready method, attempt to access the "test" control onready var. The editor crashes.

I noticed this right away in a larger project of mine after updating from beta 4 to beta 5, then created the minimal project.

### Stacktrace

```================================================================
CrashHandlerException: Program crashed
Engine version: Godot Engine v4.0.beta.custom_build (98da707df5a40d37e85070cec6babfee4fae5765)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[0] oidnUnmapBuffer
[1] oidnUnmapBuffer
[2] embree::TaskScheduler::wait
[3] embree::TaskScheduler::wait
[4] embree::TaskScheduler::wait
[5] embree::TaskScheduler::wait
[6] embree::TaskScheduler::wait
[7] embree::TaskScheduler::wait
[8] embree::TaskScheduler::wait
[9] oidnUnmapBuffer
[10] embree::TaskScheduler::wait
[11] embree::TaskScheduler::wait
[12] embree::TaskScheduler::wait
[13] embree::TaskScheduler::wait
[14] <couldn't map PC to fn name>
[15] <couldn't map PC to fn name>
[16] <couldn't map PC to fn name>
[17] <couldn't map PC to fn name>
[18] <couldn't map PC to fn name>
[19] embree::TaskScheduler::wait
[20] BaseThreadInitThunk
-- END OF BACKTRACE --
================================================================

**Comment by souplamp** [comment]
Not able to reproduce this in beta5. I was able to access the Autoload "Testui" @onready attribute in another scene and the same scene without an editor crash.

**Comment by adamscott** [comment]
Seems to be fixed on master, #67714 seems to have fixed this issue as I cannot reproduce.

**Comment by jeffgamedev** [comment]
> Seems to be fixed on master, #67714 seems to have fixed this issue as I cannot reproduce.

I had high hopes for this being resolved from recent pull requests, but I just built from the top of master and the issue still occurs.

**Comment by jeffgamedev** [comment]
This seems like a resursive/stack overflow error. Here is a log from my minimum reproduction project in the app userdata:

```Godot Engine v4.0.beta5.official.89a33d28f - https://godotengine.org
Vulkan API 1.2.0 - Using Vulkan Device #0: NVIDIA - NVIDIA GeForce RTX 2060
 
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
Control:<Control#26172457386>
```

**Comment by jeffgamedev** [comment]
I was just able to recreate this issue on an entirely different Windows PC of mine on Windows 11 using the beta 5 editor.

**Comment by jeffgamedev** [comment]
I've just built the editor from the top of master and had the crash occur. I recreated the issue in the console editor application. Here is the stacktrace:

```
Godot Engine v4.0.beta.custom_build.98da707df - https://godotengine.org
Vulkan API 1.2.0 - Using Vulkan Device #0: NVIDIA - NVIDIA GeForce RTX 2060

WARNING: FBX file import is enabled in the project settings, but no FBX2glTF path is configured in the editor settings. FBX files will not be imported.
     at: _editor_init (modules\gltf\register_types.cpp:99)

================================================================
CrashHandlerException: Program crashed
Engine version: Godot Engine v4.0.beta.custom_build (98da707df5a40d37e85070cec6babfee4fae5765)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[0] oidnUnmapBuffer
[1] oidnUnmapBuffer
[2] embree::TaskScheduler::wait
[3] embree::TaskScheduler::wait
[4] embree::TaskScheduler::wait
[5] embree::TaskScheduler::wait
[6] embree::TaskScheduler::wait
[7] embree::TaskScheduler::wait
[8] embree::TaskScheduler::wait
[9] oidnUnmapBuffer
[10] embree::TaskScheduler::wait
[11] embree::TaskScheduler::wait
[12] embree::TaskScheduler::wait
[13] embree::TaskScheduler::wait
[14] <couldn't map PC to fn name>
[15] <couldn't map PC to fn name>
[16] <couldn't map PC to fn name>
[17] <couldn't map PC to fn name>
[18] <couldn't map PC to fn name>
[19] embree::TaskScheduler::wait
[20] BaseThreadInitThunk
-- END OF BACKTRACE --
================================================================
```

**Comment by adamscott** [comment]
> In my reproduction project, simply open the "testui.gd", and in the _ready method, attempt to access the "test" control onready var. The editor crashes.

Trying to follow the instructions but it does not result into a crash. Is there something else to do?

[Peek 20-11-2022 17-58.webm](https://user-images.githubusercontent.com/270928/202931211-8b5c904b-db5f-4b54-aac8-474a5d1786db.webm)

**Comment by jeffgamedev** [comment]
> 

Nope, that is where it crashes for me frequently: just after typing "test." Thanks for trying.

**Comment by tinmanjuggernaut** [comment]
Very likely the same as https://github.com/godotengine/godot/issues/68910, caused by https://github.com/godotengine/godot/pull/68663.

See https://github.com/godotengine/godot/issues/68910#issuecomment-1322046876

**Comment by akien-mga** [comment]
Presumably fixed by #68952.
